medusa/server/api/v2/internal.py

```diff
@@ -228,7 +229,7 @@ def resource_search_indexers_for_show_name(self):
                         show['firstaired'] or 'N/A',
                         show.get('network', '').encode('utf-8') or 'N/A',
                         sanitize_filename(show['seriesname']).encode('utf-8'),
-                        show_id in all_show_ids.get(indexer, []) and (indexer, show_id)
+                        all_show_ids.get((indexer, show_id), False)
```

Comment:
The dictionary structure above makes it easy to get the key using the current search result.
If found, the value is returned, e.g. `('tvdb', 123456)`, to be used when generating the link to the current show page.
If not found, False is returned - meaning the show is not already in the library.